### DISCUSSION TITLE
can't remove agent from cli

### DISCUSSION THREAD

**Discussion by TheChaser5** [wazuh/wazuh#23093] [Q&A]
I'm want to remove an agent from my wazuh server as the pc is removed.

I want to follow this guide:
https://documentation.wazuh.com/current/user-manual/agent/agent-management/remove-agents/remove.html

but I only get this output:

root@wazuh:~# /var/ossec/bin/manage_agents

****************************************
* Wazuh v4.7.3 Agent manager.          *
* The following options are available: *
****************************************
   (I)mport key from the server (I).
   (Q)uit.
Choose your action: I or Q:

I don't get all the shown information... I'm using docker

it's the latest version and a fresh install with only 5 agents in it

**Comment by MiguelazoDS** [comment]
Hi @TheChaser5, 

If you get those options it is because you're executing the tool inside your agent, you should do that in your manager. 

- Manager options 
![image](https://github.com/wazuh/wazuh/assets/13010397/db905f82-9c4c-4ec2-a154-6c4defa27d60)

This is the agent Ubuntu Jammy
![image](https://github.com/wazuh/wazuh/assets/13010397/84993fd8-bfb7-4ebb-bc23-6083aec41fe1)

- Agent options
![image](https://github.com/wazuh/wazuh/assets/13010397/121bf0fc-4d80-49cf-aee0-f276a182baf1)

**Comment by TheChaser5** [comment]
Hello and thank you for taking the time to reply.

My wazuh ma'ager has also an agent installed. Is that what's the problem?

**Reply by MiguelazoDS** [reply]
Did you install the Wazuh agent on top of your Wazuh manager? if that is the case, yeap, that's the issue, probably the manage_agents tool in your manager was replaced. 

There's no need to install a Wazuh manager and then a Wazuh agent, the manager itself is an agent with ID "000" (you can see that using agent_control -l in your manager). 

I hope that helps.

**Reply by TheChaser5** [reply]
Hello, thank you for waiting. After removing the agent on my wazuh server it get this:
root@wazuh:~# /var/ossec/bin/manage_agents
-bash: /var/ossec/bin/manage_agents: No such file or directory

So I tought I destroyed my server somehow.
So I reverted to a backup when everyting was just installed with only one agent (windows pc)
I tried the command again but I got the same outcome.

Do I need to do someting first before entering  /var/ossec/bin/manage_agents ?

Thank you

**Reply by MiguelazoDS** [reply]
Yeah, the agent you installed on top of your manager replaced all the binaries. You should reinstall your wazuh server.

**Reply by TheChaser5** [reply]
ok thank you, I removed the agents fromm alle the servers and will reinstall the wazuh server, thank you for the help!

**Reply by MiguelazoDS** [reply]
Let us know how it goes. Regards!
